symbolic-debuginfo/src/shared.rs

```diff
@@ -0,0 +1,109 @@
+#[cfg(any(
+    feature = "macho",
+    all(
+        feature = "breakpad",
+        feature = "dwarf",
+        feature = "elf",
+        feature = "macho",
+        feature = "ms",
+        feature = "sourcebundle",
+        feature = "wasm"
+    )
+))]
```

Comment:
this expands to `macho || (... && macho && ...)`, right? In which case this is just `macho`? same below

Reply:
Yes, that is the case, I was just making it clear that `MonoArchive` is also used if all formats are enabled and the `object` module is compiled, but I'll just put it behind `macho` only.